tests/utils/apiWrappers/identity.ts

```diff
@@ -0,0 +1,143 @@
+import { Identity } from "../../../utils/apiWrappers/identity";
+require("dotenv").config({ path: ".env.test" });
+
+describe("Should test Identity object", () => {
```

Comment:
I have now way to know outside the root folder that this is a test file and I'm not sure github action make it run.

rename identity.test.js imo

Reply:
Ah correct